src/Tribe/Aggregator/Record/Queue_Processor.php

```diff
@@ -164,11 +178,23 @@ public function set_current_queue( Tribe__Events__Aggregator__Record__Queue_Inte
 	 *
 	 * If no records in need of further processing can be found it will return bool false.
 	 *
+	 * @since TBD Inclusion of a $cache param for performance purposes.
+	 *
 	 * @param boolean $interactive_only Whether or not we should look for imports that were kicked off interactively
+	 * @param boolean $cache            When checking on every request we should have a transient caching to prevent hitting the DB every time.
```

Comment:
```suggestion
	 * @param boolean $cache            When checking on every request we should utilize transient caching to prevent hitting the DB every time.
```
or "use"